modules/stanford_course_views/stanford_course_views.module

```diff
@@ -5,3 +5,27 @@
  */
 
 include_once 'stanford_course_views.features.inc';
+
+function stanford_course_views_views_pre_build(&$view) {
+  // We want to provide a link to the search view when there are no results
+
+  if (($view->name == "stanford_courses_default") || ($view->name == "courses")){
```

Comment:
Good catch Shea! 

I just pushed up the change. 


Caryl 
--- 
Caryl J. Westerberg 
Web Producer, Stanford Web Services 
cjwest@stanford.edu 

swsblog.stanford.edu 

----- Original Message -----

From: "Sherakama" <notifications@github.com> 
To: "SU-SWS/stanford_courses" <stanford_courses@noreply.github.com> 
Cc: "Caryl Westerberg" <cjwest@stanford.edu> 
Sent: Friday, February 20, 2015 9:54:42 AM 
Subject: Re: [stanford_courses] 7.x 3.x refactor view (#40) 



In modules/stanford_course_views/stanford_course_views.module : 
> @@ -5,3 +5,27 @@
>   */
>  
>  include_once 'stanford_course_views.features.inc';
> +
> +function stanford_course_views_views_pre_build(&$view) {
> +  // We want to provide a link to the search view when there are no results
> +
> +  if (($view->name == "stanford_courses_default") || ($view->name == "courses")){ 


stanford_courses_default should no longer exist. The two view machine names are: 

courses 
courses_reference 

Please check that the display names are consistent between the two. 

— 
Reply to this email directly or view it on GitHub .